e2e/specs/program-enrollment.spec.ts

```diff
@@ -75,8 +75,13 @@ test('Add and edit a program enrollment', async ({ page }) => {
   });
 
   await test.step('When I click on the `Edit` button of the created program', async () => {
-    await programsPage.overflowButton().click();
-    await programsPage.editProgramButton().click();
+    await expect(dataRow).toBeVisible();
+
+    await dataRow.hover();
+
+    await programsPage.overflowButton().click({ timeout: 10000 });
+
+    await page.getByRole('menuitem', { name: /edit/i }).click({ timeout: 10000 });
```

Comment:
Why do we need these lengthened timeouts? I'd prefer we didn't have these. 10s is not a reasonable amount of time for something to appear in the UI.